agent/structs/structs_test.go

```diff
@@ -208,6 +313,23 @@ func TestStructs_ServiceNode_PartialClone(t *testing.T) {
 	}
 }
 
+func TestStructs_ServiceNode_IsSame(t *testing.T) {
```

Comment:
This looks like copy-paste of the ToServiceNode test with the wrong name after IsSameService name change?